core/app/components/FormBuilder/ElementPanel/ComponentConfig/index.tsx

```diff
@@ -0,0 +1,61 @@
+"use client";
+
+import type { ComponentType } from "react";
+
+import dynamic from "next/dynamic";
+
+import { InputComponent } from "db/public";
+import { Skeleton } from "ui/skeleton";
+
+import type { ComponentConfigFormProps, InnerFormProps } from "./types.ts";
+
+export const ComponentConfig = ({ component, ...props }: ComponentConfigFormProps) => {
+	let ConfigForm: ComponentType<InnerFormProps>;
+	switch (component) {
+		case InputComponent.checkbox:
+			ConfigForm = dynamic(() => import("./Checkbox.tsx"), {
```

Comment:
I think there's no real way to make this less cumbersome, since template strings don't work here. LMK if there's something I'm missing though!